.travis.yml

```
@@ -1,39 +1,23 @@
+  "install":
+    [
+      "git clone https://github.com/flutter/flutter.git -b stable",
+      "./flutter/bin/flutter doctor",
+    ],
+  "script": ["./flutter/bin/flutter test"],
+  "deploy":
+    [
+      {
+        "provider": "pages",
+        "strategy": "git",
+        "on": { "branch": ["drewdev"] },
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)